react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const pokemonList = pokedex.map(pokedex =>
+<li key={pokedex.number}> {pokedex.name} </li>
+);
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+
+root.render(pokemonList);
```

Comment:
If using `li`s they should be inside a `ul` or `ol`. Even with React we want to maintain proper HTML structure. For example:
```JSX
root.render(
  <ul>
    {pokemonList}
  </ul>
);
```